### ISSUE TITLE
Is it possible to add the support of libopus and libogg to the ffmpeg build

### ISSUE THREAD

**Issue by ajithreddy** [bgrins/videoconverter.js#27]
Is it possible to add the support of libopus and libogg to the ffmpeg build.
It would be great if you can do this

**Comment by bgrins** [comment]
Yes, it would be possible.  In fact I have the basic boilerplace for libogg / libvorbis in the build script here: https://github.com/bgrins/videoconverter.js/blob/42def8c4136b03cbe55951fbf6fcda214f51059a/build/build_all_codecs.sh#L7.  If you'd like to give getting this built a shot, please do.

**Comment by duanyao** [comment]
I added libopus in PR #30, you can try `pull_source.sh` and `build.sh` in `build_edge`. FFmpeg has native support for ogg/vorbis, so I didn't add libogg/libvorbis, are they absolutely necessary?

**Comment by Kagami** [comment]
I have support for `libopus` encoder and `webm` muxer in [ffmpeg.js](https://github.com/Kagami/ffmpeg.js). I can add `ogg` muxer too if you still need it, it shouldn't increase the build size much.

**Comment by ajithreddy** [comment]
yes, that would be great, if you can do that

Thank you !!

**Comment by Kagami** [comment]
Done. Check out `ffmpeg.js@2.7.9008`
